ferrischat_webserver/src/channels/edit_channel.rs

```diff
@@ -0,0 +1,42 @@
+use actix_web::web::Json;
+
+use actix_web::{HttpRequest, HttpResponse, Responder};
+use ferrischat_common::request_json::ChannelUpdateJson;
+use ferrischat_common::types::{Channel, InternalServerErrorJson, NotFoundJson};
+
+pub async fn edit_channel(
+    req: HttpRequest,
+    channel_info: Json<ChannelUpdateJson>,
+    _: crate::Authorization,
+) -> impl Responder {
+    let channel_id = get_item_id!(req, "channel_id");
+    let bigint_channel_id = u128_to_bigdecimal!(channel_id);
+
+    let db = get_db_or_fail!();
+
+    let ChannelUpdateJson { name } = channel_info.0;
+
+    let resp = sqlx::query!(
+        "UPDATE channels SET name = $2 WHERE id = $1 RETURNING *",
+        bigint_channel_id,
+        name
```

Comment:
This ordering feels weird. Swap `$1` and `$2` in both the query and macro, please.

Reply:
fine